docs/react-testing-library/migrate-from-enzyme.mdx

```diff
@@ -258,7 +258,7 @@ it's not recommended for the reasons stated in this paragraph).
 
 We made some handy query APIs which help you to access the component elements
 very efficiently, and you can see the
-[list of available queries](dom-testing-library/api-queries.mdx) in detail.
+[list of available queries](queries/about) in detail.
```

Comment:
The link is not working on the deploy preview: https://deploy-preview-934--testing-library.netlify.app/docs/react-testing-library/migrate-from-enzyme/

Reply:
I think that's because it's missing the file extension. @Daniel-Kolev, you can try and add `.mdx` and I believe it will fix that.

Reply:
Silly me 😄 Added the extension and seems to be working on the preview now.